userguide/vms.rst

```diff
@@ -607,53 +613,64 @@ press :kbd:`Enter` again. After logging in, a
-if necessary.
-
-Download and install the Rancher system with this command:
+outside website.
 
 .. code-block:: none
 
-   sudo docker run -d --restart=unless-stopped -p 8080:8080 rancher/server
+   [rancher]@ClientHost ~]$ ping -c 3 google.com
+   PING google.com (172.217.0.78): 56 data bytes
 
+   64 bytes from 172.217.0.78: seq=0 ttl=54 time=18.613 ms
+   64 bytes from 172.217.0.78: seq=1 ttl=54 time=18.719 ms
+   64 bytes from 172.217.0.78: seq=2 ttl=54 time=18.788 ms
 
-.. note:: If the error :literal:`Cannot connect to the Docker daemon`
-   is shown, run :command:`sudo dockerd`. Then give the
-   :command:`sudo docker run` command above again.
+   --- google.com ping statistics ---
 
+   3 packets transmitted, 3 packets received, 0% packet loss
+   round-trip min/avg/max = 18.613/18.706/18.788 ms
 
-Installation time varies with processor and network connection speed,
-but typically takes a few minutes. After the process finishes and a
-command prompt is shown, type this command:
 
+If :command:`ping` returns an error, adjust the VM
+:ref:`Network Interface <vms-network-interface>` and reboot the VM.
 
-.. code-block:: none
+Download and install the Rancher server with
+:command:`sudo docker run -d --restart=unless-stopped -p 8080:8080 rancher/server`.
+
+If a :literal:`Cannot connect to the Docker daemon` error is shown,
+enter :command:`sudo dockerd` and try
+:command:`sudo docker run -d --restart=unless-stopped -p 8080:8080 rancher/server`
+again. Installation time varies with processor and network connection
+speed. :literal:`[rancher@ClientHost ~]$` is shown when the installation
+is finished.
 
-   ifconfig eth0 | grep 'inet addr'
+Enter :command:`ifconfig eth0 | grep 'inet addr'` to view the Rancher
+IP address. Enter the IP address followed by :literal:`:8080` into a web
+browser to connect to the Rancher web interface. For example, if the IP
+address is :literal:`10.231.3.208`, enter :literal:`10.231.3.208:8080`
+in the browser.
 
+The Rancher web interface takes a few minutes to start. The web browser
+might show a connection error while the web interface starts. If a
+:literal:`connection has timed out` error is shown, wait one minute and
+refresh the page.
 
-The first value is the IP address of the Rancher server. Enter the IP
-address and port :literal:`8080` as the URL in a web browser. For
-example, if the IP address was :literal:`10.231.3.208`, enter
-:literal:`10.231.3.208:8080` as the URL in the web browser.
+When the Rancher web interface loads, click :guilabel:`Add a host` from
+the banner across the top of the screen. Verify that
+:guilabel:`This site's address` is chosen and click :guilabel:`Save`.
 
-The Rancher server takes a few minutes to start. The web browser might
-show a connection error while the Rancher |web-ui| is still starting. If
-the browser shows a :literal:`connection has timed out` or a similar
-error, wait one minute and try again.
+Following the steps shown, copy the full :samp:`sudo docker run`
```

Comment:
Be specific about where the steps are shown.  Above in the docs, or on the console?

```suggestion
Following the steps shown on the console, copy the full :samp:`sudo docker run`
```